src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/TabView.xaml

```diff
@@ -701,6 +701,12 @@
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
 
+                            <!-- Uno workaround: On Android, the Height of ContentPresenters inside this Grid is not bing calculated correctly -->
+                            <!-- It can be removed once this PR is merged: https://github.com/unoplatform/uno/pull/18261 -->
+                            <Grid.RowDefinitions>
```

Comment:
Let's scope to android using XAML conditions.

Reply:
Done!